skins/laika/src/pages/membership_application.ts

```diff
@@ -25,6 +29,9 @@ interface MembershipAmountModel {
 
 const pageData = new PageDataInitializer<MembershipAmountModel>( '#app' );
 const store = createStore();
+const initialAddressType: AddressTypeModel = addressTypeFromName( pageData.applicationVars.initialFormValues.addressType );
+store.dispatch( action( NS_MEMBERSHIP_ADDRESS, setAddressType ), initialAddressType );
+store.dispatch( action( NS_MEMBERSHIP_ADDRESS, setEmail ), pageData.applicationVars.initialFormValues.email );
```

Comment:
You need to put the dispatches into a check if `pageData.applicationVars.initialFormValues` is defined, otherwise the standalone membership page won't work.